examples/ArduinoIoTCloud-Basic/arduino_secrets.h

```diff
@@ -4,12 +4,12 @@
  * https://github.com/arduino-libraries/ArduinoIoTCloud/#what
  */
 #if defined(BOARD_HAS_WIFI)
-  #define SECRET_WIFI_SSID "YOUR_WIFI_NETWORK_NAME"
-  #define SECRET_WIFI_PASS "YOUR_WIFI_PASSWORD"
+  #define SECRET_WIFI_SSID "-_-"
+  #define SECRET_WIFI_PASS "smea86861518"
```

Comment:
You forgot to remove this
```suggestion
  #define SECRET_WIFI_SSID "-_-"
  #define SECRET_WIFI_PASS "smea86861518"
```